samza-core/src/main/scala/org/apache/samza/config/JobConfig.scala

```diff
@@ -114,6 +114,10 @@ object JobConfig {
   val DEFAULT_STANDBY_TASKS_REPLICATION_FACTOR = 1
   val SYSTEM_STREAM_PARTITION_MAPPER_FACTORY = "job.system.stream.partition.mapper.factory"
 
+  // Naming format and directory for container.metadata file
+  private val CONTAINER_METADATA_FILENAME_FORMAT = "%s.metadata" // Filename: <containerID>.metadata
+  private val CONTAINER_METADATA_DIRECTORY_SYS_PROPERTY = "samza.log.dir"
```

Comment:
Is this introducing a new config or redefining the constant key for an existing? If the former, is this really required? Is this documented?

Reply:
This is already existing. This is just defining the constant.

Reply:
Why not use the existing constant then? It'll be simpler to find usages later.